src/test/resources/application.yml

```
@@ -0,0 +1,21 @@
+spring:
+  profiles:
+    active: test
+  datasource:
+    hikari:
+      url: jdbc:mysql://localhost:3306/ttbkk
```

testdb는 개발용 db와 다르다는 것을 명시하기 위해 ttbkk_test로 쓰는게 어떨까요?
```suggestion
      url: jdbc:mysql://localhost:3306/ttbkk_test
```

---

https://github.com/ttbkk-univ/ttbkk-spring/pull/32/commits/6fc1ed1dc2d9c201db70a2a234c14bd21b737c5c 수정하였습니다!